server/src/main/java/org/elasticsearch/cluster/metadata/Metadata.java

```diff
@@ -550,6 +550,18 @@ public int getTotalNumberOfIndices() {
         return indexCount;
     }
 
+    /**
+     * @return {code true} if there are any indices in any project in this cluster
+     */
+    public boolean hasAnyIndices() {
+        for (ProjectMetadata project : projects().values()) {
+            if (project.indices().isEmpty() == false) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
I used `getTotalNumberOfIndices` when merging it to the internal branch. This new method can be more efficient when there are actually multiple projects.

Reply:
Yes, efficiency was my concern - iterating through all the projects isn't the worst thing we do, but it's totally unnecessary for this case.